ansible/roles/atmo-ssh-setup/tasks/ssh-setup.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Directly check linux distribution
```

Comment:
This task is a little wonky (on Ubuntu that file won't be there) but I understand what you're doing.

An alternative approach is running the [setup](http://docs.ansible.com/ansible/setup_module.html) module (which discovers the actual distro), and then gather OS-specific variables [this way](https://github.com/c-mart/atmosphere-ansible/blob/c10837781229281f11606a5d859c6e3ad7f41936/ansible/roles/atmo-dhcp/tasks/main.yml#L8). (At this point in the role you should be able to gather facts, as you already have a working Python 2 interpreter and you know which user you can log in as.)